temporalcli/commands.gen.go

```diff
@@ -3506,9 +3506,9 @@ func NewTemporalWorkflowListCommand(cctx *CommandContext, parent *TemporalWorkfl
 	s.Command.Use = "list [flags]"
 	s.Command.Short = "Show Workflow Executions"
 	if hasHighlighting {
-		s.Command.Long = "List Workflow Executions. The optional \x1b[1m--query\x1b[0m limits the output to\nWorkflows matching a Query:\n\n\x1b[1mtemporal workflow list \\\n    --query YourQuery\x1b[1m\x1b[0m\n\nVisit https://docs.temporal.io/visibility to read more about Search Attributes\nand Query creation. See \x1b[0mtemporal batch --help` for a quick reference.\n\nView a list of archived Workflow Executions:\n\n\x1b[1mtemporal workflow list \\\n    --archived\x1b[0m"
+		s.Command.Long = "List Workflow Executions. The optional \x1b[1m--query\x1b[0m limits the output to\nWorkflows matching a Query:\n\n\x1b[1mtemporal workflow list \\\n    --query YourQuery\x1b[0m\n\nVisit https://docs.temporal.io/visibility to read more about Search Attributes\nand Query creation. See \x1b[1mtemporal batch --help\x1b[0m for a quick reference.\n\nView a list of archived Workflow Executions:\n\n\x1b[1mtemporal workflow list \\\n    --archived\x1b[0m"
```

Comment:
Anyone know why the ANSI escapes changed here? Haven't tested in a terminal but the new version LGTM. (`1m` is a transition to bold and `0m` is a transition that removes all flags.)

Reply:
I believe this is related to [this change](https://github.com/temporalio/cli/pull/863/files#diff-1bf9d89c09230b7fb737937c657147b516bac4d39fa100086a30871c7abad39eL3668); previously there was an extraneous hanging backtick on the code block for `temporal workflow list --query YourQuery`, so the parser that creates the highlighted version of the help text may have gotten a bit confused. Here's how it looks today: 

<img width="590" height="128" alt="image" src="https://github.com/user-attachments/assets/91b02988-73f1-4d07-a4fb-e00359a5642d" />

With this patch:

<img width="577" height="125" alt="image" src="https://github.com/user-attachments/assets/9c8f4795-07a8-4221-8cb5-eea52389f7bf" />

This is actually the exact typo that I noticed that sent me down this rabbit hole 😄

Reply:
Excellent, thanks for looking into this.